Modules/CMakeLists.txt

```diff
@@ -6,3 +6,4 @@ add_subdirectory(Example)
 add_subdirectory(Skeleton)
 add_subdirectory(TOF)
 add_subdirectory(EMCAL)
+
```

Comment:
With this newline, o2-qc-module-configurator.sh will add a next module after an empty line (as below), could you delete it please?
```
...
add_subdirectory(EMCAL)

add_subdirectory(NewDetector)
```

Reply:
thank you for spotting this !